File: fabric/src/main/java/juuxel/adorn/client/resources/BookManagerFabric.java
package juuxel.adorn.client.resources;

import juuxel.adorn.client.book.BookManager;
import net.fabricmc.fabric.api.resource.IdentifiableResourceReloadListener;
import net.minecraft.util.Identifier;

public final class BookManagerFabric extends BookManager implements IdentifiableResourceReloadListener {
    public static final BookManagerFabric INSTANCE = new BookManagerFabric();

    @Override
    public Identifier getFabricId() {
        return ID;
    }
}

File: fabric/src/main/kotlin/juuxel/adorn/resources/BookManagerFabric.kt
package juuxel.adorn.resources

import juuxel.adorn.AdornCommon
import juuxel.adorn.client.book.BookManager
import net.fabricmc.fabric.api.resource.IdentifiableResourceReloadListener
import net.minecraft.util.Identifier

object BookManagerFabric : BookManager(), IdentifiableResourceReloadListener {
    private val ID = AdornCommon.id("book_manager")

    override fun getFabricId(): Identifier = ID
}
